import/import_planet.sh

```diff
@@ -79,20 +80,27 @@ if [[ ! -x $OSM2PGSQL ]]; then
 fi
 
 # if there's no planet, then download it
-PLANET="planet-${PLANET_DATE}.osm.pbf"
-if [[ ! -f "planet/${PLANET}" ]]; then
+if [[ ! -f "planet/${PLANET_FILE}" ]]; then
     echo "downloading planet" > $STATUS
     rm -rf planet
     mkdir planet
     cd planet/
-    wget -q "http://s3.amazonaws.com/osm-pds/${PLANET_YEAR}/${PLANET}"
-    wget -q "http://s3.amazonaws.com/osm-pds/${PLANET_YEAR}/${PLANET}.md5"
+    wget -q "${PLANET_URL}"
+
+    if [[ -n "${PLANET_MD5_URL}" ]]; then
+        wget -q "${PLANET_MD5_URL}"
+    fi
+
     cd ..
 fi
 
-# always check the planet checksum!
-echo "checking planet file MD5" > $STATUS
-(cd planet && md5sum --check "${PLANET}.md5")
+# check the md5sum if the md5 file was downloaded
+if [[ -f "planet/${PLANET_FILE}.md5" ]]; then
```

Comment:
Does this assume that `${PLANET_MD5_URL} = "${PLANET_URL}.md5"`? Consequences aren't bad if not - it just skips the check, and it's a very strong convention to have the MD5 file be the original file + `.md5`. Maybe line 91 above could be `wget -O "${PLANET_FILE}.md5" -q "${PLANET_MD5_URL}"` to cover this (microscopic) corner case?

Reply:
Yea, good catch. I'll adjust this in the next PR.

Reply:
Resolved in 273da15.